src/main/java/controller/CalculatorController.java

```diff
@@ -0,0 +1,30 @@
+package controller;
+
+import domain.StringCalculator;
+import view.InputView;
+import view.OutputView;
+
+import java.util.List;
+
+public class CalculatorController {
+
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public CalculatorController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void operate() {
+        String input = inputView.inputString();
+        StringCalculator stringCalculator = new StringCalculator(input);
+
+        List<String> strNumber;
+        strNumber = stringCalculator.checkSplit();
```

Comment:
```suggestion
        List<String> strNumber = stringCalculator.checkSplit();
```

strNumber를 조금 더 이름을 풀어서 쓰면 좋을 것 같아요 자료형은 네이밍에 지양하고요!